### PR TITLE
T16673

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T16673

### CODE REVIEW

**Comment by tchx84** [comment]
Merged after https://github.com/endlessm/eos-knowledge-lib/pull/1627

**Comment by ptomato** [comment]
This seems fine to me, but I'd squash the last three commits into one.

File: tests/js/app/testBuffetHistoryStore.js [hunk 1]
```diff
@@ -1,163 +0,0 @@
...
-    });
-
-    describe('when an article card is clicked', function () {
-        beforeEach(function () {
-            let model = new DModel.Article();
-
-            dispatcher.dispatch({
-                action_type: Actions.ITEM_CLICKED,
-                model: model,
-                context: [model],
```
Comment (ptomato) [inline tests/js/app/testBuffetHistoryStore.js]: I would suggest adding these context parameters to the corresponding dispatch actions in testMeshHistoryStore.js.

**Comment by ebassi** [comment]
👍

**Pull request merged**